vite.config.js

```diff
@@ -7,10 +7,10 @@ export default defineConfig({
     outDir: join(__dirname, "inst/www/react-tools/"),
     lib: {
       // Could also be a dictionary or array of multiple entry points
-      entry: resolve(__dirname, '/srcjs/react-tools.js'),
+      entry: resolve(__dirname, 'srcjs/react-tools.js'),
```

Comment:
Leading slash was causing issues when I tried to build

Reply:
Works without leading slash also on my two setups.